config/docker-compose.yaml

```diff
@@ -47,18 +60,31 @@ services:
       CHROME_PATH: /usr/lib/chromium/
     profiles:
       - be
-    ports:
-      - 3000:3000
+      - prox
+    #ports:
+    #  - 3000:3000
+    labels:
+      - "traefik.http.routers.backend.rule=PathPrefix(`/api`)||PathPrefix(`/explorer`)||PathPrefix(`/users`)||PathPrefix(`/locations`)"
```

Comment:
through adding `/api` to the `lbBaseURL` there shouln't be a need to add `||PathPrefix(`/users`)||PathPrefix(`/locations`)` here anymore...